UnitsNet/UnitDefinitions/ReactivePower.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Name": "ReactivePower",
+  "BaseUnit": "VoltAmpereReactive",
```

Comment:
We have previously gone lower-case for words bound by hyphen `-`.
So `VoltampereReactive`.